README.md

```diff
@@ -39,10 +38,25 @@ layer.addTo(map);
 map.fitBounds(layer.getBounds());
 ```
 
-# advanced usage
+The following options are supported:
+
+| **Option**                   | **Data Type** | **Default value**                          | **Description**                                                                                                                                                                                     |
+|------------------------------|---------------|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| baseUrl                      | string        | `href` of the `self` link in the STAC data | The base URL for relative links. Should be provided if the STAC data has no self link and links are not absolute. |
+| buildUrlTemplate             | function      | undefined                                  | For server-side rendering of imagery. See the chapter [using a tiler](#using-a-tiler) and [buildTileUrlTemplate](#buildTileUrlTemplate) for details. |
+| displayPreview               | boolean       | false                                      | Allow to display the asset with role `thumbnail` or the link with relation type `preview`. The previews are usually not covering the full extents and as such may be placed incorrectly on the map. |
+| displayOverview              | boolean       | true                                       | Allow to display the asset with role `overview` or `visual`. |
```

Comment:
The code currently doesn't require displayOverview to be true to display assets with the "visual" key:
https://github.com/stac-utils/stac-layer/blob/169883c0c49b5c17d1a1aaa863eee2f2dd925f19/src/index.js#L361

I'd recommend updating this to "Allow to display the asset with role `overview`".  Or we can update the code from:
```if (addedImagery === false && hasAsset(assets, "visual")) {```
to
```if (addedImagery === false && displayOverview && hasAsset(assets, "visual")) {```

Let me know your preference :-)

Reply:
As "visual" is not standardized, I see it as a "fallback" for "overview. So I think I'd prefer the option to add the displayOverview check into the if clause.

Reply:
@m-mohr , sounds good.  I just updated the code to reflect this: https://github.com/stac-utils/stac-layer/commit/ed80a5eb498d228318adfbf7fc089a6d812e7a57

Reply:
Great, thanks a lot!